app/javascript/common/formUtilities/validation.js

```diff
@@ -0,0 +1,23 @@
+export const required = value => (value ? undefined : 'Required');
+
```

Comment:
Should those be extracted?

Reply:
We won't need this code once we start using `redux-form-validators`